test-case/check-kmod-load-unload.sh

```diff
@@ -89,6 +89,9 @@ do
     ## - 2a: check for errors after insertion
     dlogi "checking for general errors after kmod insert with sof-kernel-log-check tool"
     sof-kernel-log-check.sh $KERNEL_LAST_LINE
+    [[ $? -ne 0 ]] && \
+        dloge "Found error(s) in kernel log after module insertion" && \
```

Comment:
no need to new line here. This look worse then a long line.

Reply:
replace with if else, and align code style of the whole file